src/test/spec/unified_runner/operation/search_index.rs

```diff
@@ -0,0 +1,151 @@
+use bson::{to_bson, Bson, Document};
```

Comment:
good call on moving these to their own file, `operation.rs` is huge and could probably use some modularization

Reply:
Filed RUST-1800 for this.